src/routes/safes/entities/safe-info.entity.ts

```diff
@@ -20,11 +20,11 @@ export class SafeState {
   readonly owners: AddressInfo[];
   @ApiProperty()
   readonly implementation: AddressInfo;
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: Array<AddressInfo>, nullable: true })
```

Comment:
This won't work: you'd need to set it as `type: [AddressInfo]` or with `type: AddressInfo, isArray: true`

Reply:
Fixed in [1b99b95](https://github.com/safe-global/safe-client-gateway/pull/765/commits/1b99b95ba4cc5b36f637daf9fdec1f88afb15e68), had to rebase to include https://github.com/safe-global/safe-client-gateway/pull/766 and test it. Sorry about the back and forth 🙂